Makefile.am

```diff
@@ -69,6 +69,9 @@ nginxconfig_DATA = etc/nginx/coolwsd.conf
 
 ACLOCAL_AMFLAGS = -I m4
 
+DIST_FOLDER ?= $(abs_builddir)/browser/dist
```

Comment:
Not clear why we export this here, when we can calculate this from abs_top_builddir in the test/Makefile.am surely and not jam stuff into the environment (?) =)

Good fix though, can you simplify & push with that ?

Thanks!

Reply:
We have the same variable in browser/Makefile.am, it is configurable to custom dist folder, I cannot do it in
test/Makefile.am since browser project can have different value.

Reply:
$(abs_top_builddir)/browser is not reliable ? - note the _top_ in the variable name (?) =)

Reply:
Yes, please check browser/Makefile.am:9

DIST_FOLDER ?= $(builddir)/dist

This variable can be customized, if not set default value